server/app/com/fortysevendeg/exercises/ApplicationLoader.scala

```diff
@@ -62,10 +60,6 @@ class Components(context: Context)
 
   implicit val wsClient: WSClient = NingWSClient()
 
-  // touch lazy val to enable
-  applicationEvolutions
-  override def dynamicEvolutions: DynamicEvolutions = new DynamicEvolutions
```

Comment:
Can we check here if the app is in DEV mode and then run them only in such case?